<ui version="4.0" >
 <class>DeckProperties</class>
 <widget class="QDialog" name="DeckProperties" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit deck properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="qtabwidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Description &amp;&amp; Synchronisation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>&lt;h1>Deck description&lt;/h1></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="deckDescription" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>&lt;h1>Synchronisation&lt;/h1>Synchronisation lets you use your deck on multiple computers at the same time. You can also use it to study on the web, for when you're not at home. You can also study on your mobile phone if your phone has internet access.</string>
         </property>
         <property name="scaledContents" >
          <bool>false</bool>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="doSync" >
         <property name="text" >
          <string>Synchronize this deck with name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="syncName" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Priorities &amp;&amp; Postponing</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_21" >
         <property name="text" >
          <string>&lt;h1>Very high priority&lt;/h1>A comma-separated list of tags to prioritize. Matching cards are placed at the top of the revision queue.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="highPriority" />
       </item>
       <item>
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>&lt;h1>High priority&lt;/h1>Matching cards are placed at the top of the revision queue or new card queue, depending on if they have been seen before.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="medPriority" />
       </item>
       <item>
        <widget class="QLabel" name="label_24" >
         <property name="text" >
          <string>&lt;h1>Low priority&lt;/h1>Matching cards are placed at the bottom of the new card queue.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lowPriority" />
       </item>
       <item>
        <widget class="QLabel" name="label_23" >
         <property name="text" >
          <string>&lt;h1>Postponing&lt;/h1>Any matching cards will not be shown until unsuspended. The web and phone interfaces automatically add 'noweb' and 'nophone' - give these tags to your cards to prevent them from showing on the phone or web interface.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="postponing" />
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Scheduling</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_15" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;h1>New &amp; young cards&lt;/h1>The initial times to use when scheduling cards, in days.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="minimumSize" >
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;b>Hard&lt;/b></string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QLineEdit" name="easyMin" />
         </item>
         <item row="1" column="1" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" >
          <widget class="QLineEdit" name="midMax" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>&lt;b>Easy&lt;/b></string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>&lt;b>Medium&lt;/b></string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" >
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QLineEdit" name="hardMin" />
         </item>
         <item row="1" column="2" >
          <widget class="QLineEdit" name="midMin" />
         </item>
         <item row="0" column="3" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4" >
          <widget class="QLineEdit" name="easyMax" />
         </item>
         <item row="0" column="4" >
          <widget class="QLineEdit" name="hardMax" />
         </item>
         <item row="0" column="1" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Max</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>The order to show new cards.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QComboBox" name="newCardOrder" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_18" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;h1>Delay on mistake&lt;/h1>The time until Anki shows you a card you got wrong. The default is 10 minutes.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_20" >
           <property name="text" >
            <string>&lt;b>Totally forgot (0)&lt;/b></string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_22" >
           <property name="text" >
            <string>&lt;b>Made a mistake (1) on a young card</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" >
          <widget class="QLineEdit" name="delay0" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLineEdit" name="delay1" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_19" >
           <property name="text" >
            <string>&lt;b>Made a mistake (1) on a mature card&lt;/b></string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2" >
          <widget class="QLineEdit" name="delay2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_25" >
         <property name="text" >
          <string>&lt;h1>Final drill &amp; failed card limit&lt;/h1></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_26" >
           <property name="text" >
            <string>The number of hours to collapse in the final drill. Use 0 to disable.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" >
          <widget class="QLineEdit" name="collapse" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_12" >
           <property name="text" >
            <string>The maximum number of failed cards to allow at once.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLineEdit" name="failedCardMax" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Models</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>&lt;h1>Models&lt;/h1>Models define what sort of information you're studying, and the way to show it to you. Models range from a simple representation of a single flashcard with a "front" and "back" side, to more complicated domain specific models: a model for Russian verbs may define two verb forms, and test you on both of them.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="modelsList" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="modelsAdd" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modelsEdit" >
           <property name="text" >
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modelsDelete" >
           <property name="text" >
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qtabwidget</tabstop>
  <tabstop>deckDescription</tabstop>
  <tabstop>doSync</tabstop>
  <tabstop>syncName</tabstop>
  <tabstop>highPriority</tabstop>
  <tabstop>medPriority</tabstop>
  <tabstop>lowPriority</tabstop>
  <tabstop>postponing</tabstop>
  <tabstop>hardMin</tabstop>
  <tabstop>hardMax</tabstop>
  <tabstop>midMin</tabstop>
  <tabstop>midMax</tabstop>
  <tabstop>easyMin</tabstop>
  <tabstop>easyMax</tabstop>
  <tabstop>newCardOrder</tabstop>
  <tabstop>delay0</tabstop>
  <tabstop>delay1</tabstop>
  <tabstop>delay2</tabstop>
  <tabstop>collapse</tabstop>
  <tabstop>failedCardMax</tabstop>
  <tabstop>modelsList</tabstop>
  <tabstop>modelsAdd</tabstop>
  <tabstop>modelsEdit</tabstop>
  <tabstop>modelsDelete</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeckProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeckProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>334</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doSync</sender>
   <signal>toggled(bool)</signal>
   <receiver>syncName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel" >
     <x>327</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
